pkg/test/testdata/testdata/gen.go

```diff
@@ -9,5 +9,8 @@
 //go:generate cert "Test Cortex CA" cortex/root.crt cortex/root.key --profile=intermediate-ca --ca=root_ca.crt --ca-key=root_ca.key
 //go:generate cert "Test Cortex Client" cortex/client.crt cortex/client.key --profile=leaf --ca=cortex/root.crt --ca-key=cortex/root.key --san=localhost --san=127.0.0.1
 //go:generate cert "Test Cortex Server" cortex/server.crt cortex/server.key --profile=leaf --ca=cortex/root.crt --ca-key=cortex/root.key --san=localhost --san=127.0.0.1
+//go:generate cert "Test Alerting CA" alerting/root.crt alerting/root.key --profile=intermediate-ca --ca=root_ca.crt --ca-key=root_ca.key
+//go:generate cert "Test Alerting Client" alerting/client.crt alerting/client.key --profile=leaf --ca=cortex/root.crt --ca-key=cortex/root.key --san=localhost --san=127.0.0.1
+//go:generate cert "Test Alerting Server" alerting/server.crt alerting/server.key --profile=leaf --ca=cortex/root.crt --ca-key=cortex/root.key --san=localhost --san=127.0.0.1
```

Comment:
should this be:
```suggestion
//go:generate cert "Test Alerting Client" alerting/client.crt alerting/client.key --profile=leaf --ca=alerting/root.crt --ca-key=alerting/root.key --san=localhost --san=127.0.0.1
//go:generate cert "Test Alerting Server" alerting/server.crt alerting/server.key --profile=leaf --ca=alerting/root.crt --ca-key=alerting/root.key --san=localhost --san=127.0.0.1
```
?

Reply:
yes